src/base/moduledownloader.lua

```diff
@@ -0,0 +1,125 @@
+--
+-- moduledownloader.lua
+-- Downloads a module from a package server
+-- Copyright (c) 2002-2016 Jason Perkins and the Premake project
+--
+
+
+---
+-- progress bar.
+--
+	local function _http_progress(total, current)
```

Comment:
yeah I noticed the same problem on our Travis builds. A simple solution I can think of is maybe disable it if a `CI` environment variable exists.